.github/workflows/README.md

```diff
@@ -1,12 +1,27 @@
 Actions:
 
+You have an automerge action which has not been explained. Does this allow you to merge to automatically when the ci completes successfully? What is the purpose of this action?
+Also what is this line doing ` uses: "pascalgn/automerge-action@v0.9.0"`? Probably want to include a link https://github.com/pascalgn/automerge-action
```

Comment:
Read how it work here: https://github.com/marketplace/actions/merge-pull-requests, we do nothing special.